packages/store/tests/select.spec.ts

```diff
@@ -68,6 +84,70 @@ describe('Select', () => {
 
   const states = [MySubState, MySubSubState, MyState];
 
+  it('should be throw error when forgot to import the ngxs module', () => {
```

Comment:
`should throw an exception when the user has forgotten to import the NGXS module`

Reply:
done